### DISCUSSION TITLE
Adding our own AI upscale models

### DISCUSSION THREAD

**Discussion by jhitz0z** [chaiNNer-org/chaiNNer#2994] [Ideas]
Is there a way we add and run our own AI .pth models, instead of the current preset models running on this application and the other applications? I'm currently using the windows version of the applications but there is no source file directory.

**Comment by joeyballentine** [comment]
chaiNNer does not come with preset pth models. The load image node has a file picker that you use to select models from anywhere on your computer. So, I'm not sure what exactly you're running into here.

**Comment by joeyballentine** [comment]
*Load model node, not load image node, my bad
